tools/ZAPD/ZAPD/ZCollision.cpp

```diff
@@ -61,18 +61,22 @@ void ZCollisionHeader::ParseRawData()
 		currentPtr += vec.GetRawDataSize();
 		vertices.push_back(vec);
 	}
-
+	
```

Comment:
Looks like this needed formatting, but not going to worry about that for a subrepo.